Library/Homebrew/development_tools.rb

```diff
@@ -99,6 +99,10 @@ def curl_handles_most_https_certificates?
     def subversion_handles_most_https_certificates?
       true
     end
+
+    def keep_file(dir)
+      dir/".keepme"
+    end
```

Comment:
Can kill this method: https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)